source/_includes/playground/header.liquid

```diff
@@ -18,7 +18,16 @@
         <button
           type="button"
           id="playground-copy-url"
-          class="sl-c-button">Copy URL</button>
+          class="sl-c-button sl-c-button--copy sl-c-button--icon-text">
+          <div class="sl-c-playground-button--copy--copy">
+            {%- render 'icons/copy' -%}
+          </div>
```

Comment:
I removed the "Click to Copy" screen reader text here since the icon is paired with actual text in the button that says "button, Copy URL" so it felt unnecessary here.